lldb/include/lldb/API/SBCommandReturnObject.h

```diff
@@ -42,6 +42,8 @@ class LLDB_API SBCommandReturnObject {
 
   bool IsValid() const;
 
+  const char *GetCommand();
```

Comment:
The docs currently in CommandReturnObject.h should certainly be here (or in both places)...